support-internationalisation/build.sbt

```diff
@@ -2,8 +2,8 @@ import sbtrelease.ReleaseStateTransformations._
 
 name := "support-internationalisation"
 
-scalaVersion := "3.2.2" // beware >= 3.1.3 has an incompatibility with <= 2.13.8 which causes compile issue `Unsupported Scala 3 generic tuple type scala.Tuple in bounds of type MirroredElemTypes; found in  scala.deriving.Mirror.<refinement>.`
-crossScalaVersions := Seq("2.13.8", "3.2.2")
+scalaVersion := "3.3.3"
```

Comment:
comment no longer needed since the days of https://github.com/scala/scala3/issues/15878 are passed